bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "legislative-simulation",
+  "version": "0.0.0",
+  "homepage": "https://github.com/jtoney1/legislative-simulation",
+  "authors": [
+    "james <jatoney@hotmail.com>"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```

Comment:
I added those with a CDN via the index.html.  I just did the latest version I think...  If we are using Polymer (Which I would like to) I think the webcomponents.js polyfill only supports ie10+  ( I haven't really investigated this, so maybe there is a way around it)